app/helpers/admin/users_helper.rb

```diff
@@ -1,2 +1,67 @@
+require 'user_statistic'
 module Admin::UsersHelper
+  include Statistic::UserStatistic
+
+  def comments_graph(comments)
+    require 'rubyvis'
+    comments_data = comment_statistic_graph_data(comments)
+    days = last_days
+
+
+#/* Sizing and scales. *
+    w = 600
+    h = 250
+    x = pv.Scale.linear(0, comments_data.max).range(0, w)
+    y = pv.Scale.ordinal(pv.range(comments_data.count)).split_banded(0, h, 4/5.0)
+
+#/* The root panel. */
+    vis = pv.Panel.new()
+              .width(w)
+              .height(h)
+              .bottom(20)
+              .left(60)
+              .right(10)
+              .top(5);
```

Comment:
what here doing the semicolumn?

Reply:
i have got it from example(rubyvis gem examples at github page), a then I tried to change value i change something like 'padding top' to that gpaph